extern/boostd-data/couchbase/db.go

```diff
@@ -665,3 +666,155 @@ func serviceName(svc gocb.ServiceType) string {
 	}
 	return "unknown"
 }
+
+// RemoveMetadata
+func (db *DB) RemoveMetadata(ctx context.Context, pieceCid cid.Cid) error {
+	ctx, span := tracing.Tracer.Start(ctx, "db.remove_metadata")
+	defer span.End()
+
+	return db.withCasRetry("remove-metadata-for-piece", func() error {
+		var getResult *gocb.GetResult
+		k := toCouchKey(pieceCid.String())
+		getResult, err := db.pcidToMeta.Get(k, &gocb.GetOptions{Context: ctx})
+		if err != nil {
+			return fmt.Errorf("getting piece cid to metadata for piece %s: %w", pieceCid, err)
+		}
```

Comment:
Here we should check if the error is a not found error. If so, we should just return nil.

Reply:
This is not resolved